wallet-webapps-common/src/main/webapp/vue-app/components/WalletNotificationAlert.vue

```diff
@@ -8,7 +8,17 @@
     app>
     <exo-notification-alert
       :alert="alert"
-      @dismissed="clear" />
+      @dismissed="clear">
+      <template #actions>
+        <a
+          :href="transactionHashLink"
+          :title="$t('exoplatform.wallet.message.transactionExplorerLink')"
+          rel="external nofollow noreferrer noopener"
+          target="_blank">
+          <br>{{ transactionLinkLabel }}
```

Comment:
```suggestion
          {{ transactionLinkLabel }}
```
Why using the `br` element, it's deprecated in fact and do we need a new line for link display ?